js/app.js

```diff
@@ -159,53 +288,34 @@ function createResultsChart() {
   });
 }
 
-// Event handler for clicking on an image
-function handleProductClick(event) {
-  round++;
-  let targetIndex = productArrayIndices[event.target.id];
-  productsArray[targetIndex].clicks++;
-
-  if (round === TOTAL_ROUNDS) {
-    // When total rounds have passed, visually style the view results button to indicate a change of state
-    document.getElementById('resultsButton').style.border = '0.25rem solid #c7a46e';
-    for (let img of document.getElementsByClassName('productImage')) {
-      img.removeEventListener('click', handleProductClick);
-    }
-    return;
+// Event handler for resetting the survey process
+function resetSurvey() {
+  localStorage.clear();
+  // Use canvas.js getChart method to see if the chart has been rendered in the resultsChart canvas and, if so, destroy it
+  let chartStatus = Chart.getChart('resultsChart');
+  if (chartStatus !== undefined) {
+    chartStatus.destroy();
```

Comment:
I like that you put this on a button instead of assuming it was the desired behavior!